common/app/views/fragments/email/stylesheets/connectedStyleOverride.scala.html

```diff
@@ -29,7 +29,8 @@
     .tone-news .headline {
         color: #333333;
     }
-    .tone-news .fc-item__kicker {
+    .tone-news .fc-item__kicker,
+    .tone-news .byline {
```

Comment:
Yep! Almost all the AMP styling is not in scss. This PR is a good starting point: https://github.com/guardian/frontend/pull/15154/files